src/api/controller/AuthenticationController.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express";
+import { IUserDatabase } from "../../database/IUserDatabase";
+import { Validator } from "../../utils/Validator";
+import { UserSchema } from "../model/user/UserSchema";
+
+export class AuthenticationController {
+    private database: IUserDatabase;
+
+    constructor(database: IUserDatabase) {
+        this.database = database;
+    }
+
+    login = async (req: Request, res: Response) => {
+        let existingUser : any = await this.database.GetUser(new Map([["uid", req.uid!]]));
+
+        if (existingUser === null)
+            res.status(400).json({ success: false, data: `user doesn't exists` });
+        else 
+            res.redirect(302, `/api/user/${existingUser._id}`);
```

Comment:
Let's follow the google JS [style guide](https://google.github.io/styleguide/jsguide.html), and [add braces for all control structures](https://google.github.io/styleguide/jsguide.html#formatting-braces-all).

Reply:
I hate their style. It's not concise at all. I will fix it though.